README.md

```diff
@@ -1,5 +1,89 @@
+    - `total_count: Number` (The total amount of objects that the paginator received)
+    - `previous_page: Number` (The number of the previous page. Will be null if there's nothing to show)
+    - `current_page: Number` (The number of the current page that is being shown)
+    - `next_page: Number` (The number of the next page. Will be null if there's nothing to show)
+    - `previous_page_url: String url` (A string url that leads to the previous page. Will be null if there's nothing to show)
+    - `next_page_url: String url` (A string url that leads to the next page. Will be null if there's nothing to show)
+
+
+## Usage
+After installing the package, all you have to do is require it inside the file were you want to paginate something:
+```js
+const nodePagination = require('@wolox/pagination-node');
+```
+Then, when you have the collection you want to paginate, you can make use of the `paginate` method of the package in any of the following ways
+
+* Without specifying options (default options values will be used):
+```js
+const paginatedResponse = nodePagination.paginate(collection, request, {});
+```
+
+* Specifying the `page` option (as `limit` option is missing, the limit value will be the default one):
+```js
+const paginatedResponse = nodePagination.paginate(collection, request, { page: 2 });
+```
+
+* Specifying the `limit` option (as `page` option is missing, the page value will be the default one):
+```js
+const paginatedResponse = nodePagination.paginate(collection, request, { limit: 5 });
+```
+
+* Specifying both `page` and `limit` values:
+```js
+const paginatedResponse = nodePagination.paginate(collection, request, { page: 2, limit: 5 });
+```
+
+`collection`: This parameter could also be named as `content` and it represents the group of objects you want to paginate. This must be an Array, otherwise the `paginate` method will throw an error.
+
+`request`: Used to concat a string url that leads to the previous and next pages. This should be an `IncomingMessage` object given that it uses the `request.headers.host` and `request.url` attributes to arrange a valid url string, otherwise those urls will return null.
+
+`options`: An object containing options to modify the requested page. So far, only `page` and `limit` options are allowed.
+
+
+### Default option values
+* `page`: 1
+* `limit`: 25
+
+### Modifying default params
+If you wanted to stablish different option values as defaults all you have to do is set the needed value to the corresponding option like this:
+```js
+nodePagination.defaultLimit = NEW_LIMIT;
+nodePagination.defaultPage = NEW_PAGE;
+```
+It's recommended to do this on a setup/startup step of the applicacion that requires the tool, for example requiring it and changing its default values in a script that runs before a controller rendering the result using the tool itself.
```

Comment:
What about changing it to "...and changing It's default values in a middleware that runs before the controller action that utilizes this tool"